Assets/UnityIO.cs

```diff
@@ -0,0 +1,49 @@
+using UnityEngine;
+using System;
+using System.IO;
+using System.Collections;
+
+public delegate void UnityIORequestCallback(object userData, Response response);
```

Comment:
We might be able to reduce the visibility by leaving that within the `UnityIO` class scope.